pkg/action/env.container.go

```diff
@@ -223,6 +242,19 @@ func (c *containerEnv) Close() error {
 	return c.driver.Close()
 }
 
+func (c *containerEnv) ImageRemove(ctx context.Context, a *Action) {
+	resp, err := c.driver.ImageRemove(ctx, a.ActionDef().Image, types.ImageRemoveOptions{
+		Force:         false,
+		PruneChildren: false,
+	})
+
+	if err != nil {
```

Comment:
Write here error handling, but return the error. The messages must be output by the caller.